components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -501,7 +504,7 @@ public User updateUser(User user, Map<String, Boolean> requiredAttributes) throw
 
         //Handle single attribute search
```

Comment:
Shall we add a space in the comment, after `//` ?

Reply:
Fixed: dc26889c2156cbd827075fa28e80cddd63dc87c4